core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -130,6 +130,12 @@ enum SelectAliasLookup {
    */
   SelectAliasLookup isSelectAlias();
 
+  /**
+   * Allow non-GROUP BY columns in SELECT (MySQL loose mode).
+   * If enabled, wrap with ANY_VALUE().
```

Comment:
I don't think that SqlConformance should dictate how this is implemented.
The second line should not be here.